pfcon/pfcon.py

```diff
@@ -116,16 +118,30 @@
         "openshiftlocal": {
             "compute": {
                 "addr":                         "pman-myproject.127.0.0.1.nip.io",
-                "baseURLpath":                  "api/v1/cmd/",
-                "allowUnverifiedCertificates":  False,  
+                "baseURLpath":                  "api/v1/cmd/", 
                 "status":                       "undefined"
             },
             "data": {
                 "addr":                         "pfioh-myproject.127.0.0.1.nip.io",
                 "baseURLpath":                  "api/v1/cmd/",
-                "allowUnverifiedCertificates":  False,  
                 "status":                       "undefined"
             }
+        },
+        "openshiftlocal-tokens": {
```

Comment:
@betaredux Why is this here?  Is this just here for testing?

Reply:
It doesn't do anything as far as I can tell. It would probably be better to ask @iamemilio 
(Also, you misspelled my username).

Reply:
Hi, this was just for testing pfcon against pfioh with auth enabled in openshift. It is fine to remove if you would like.